neurogym/wrappers/monitor.py

```diff
@@ -58,11 +75,16 @@ def __init__(
             self.t = 0
         self.verbose = verbose
         if folder is None:
-            # FIXME is it ok to use tempfile.TemporaryDirectory instead or does this need to be stored locally always?
-            self.folder = "tmp"
+            temp_dir = tempfile.mkdtemp(prefix="neurogym_monitor_")
+            self.folder = temp_dir
+        else:
+            self.folder = str(folder)
         Path(self.folder).mkdir(parents=True, exist_ok=True)
-        # seeding
-        self.sv_name = self.folder + self.env.__class__.__name__ + "_bhvr_data_" + name + "_"  # FIXME: use pathlib
+        self.name = name
+
+        # Create save name using pathlib for cross-platform compatibility
+        self.sv_name = Path(self.folder) / f"{self.env.unwrapped.__class__.__name__}_bhvr_data_{name}_"
+
```

Comment:
I put this FIXME in a while ago, but am actually not sure that is what we want. I actually think it's better to create a normal folder after all, so the user can see it (if using a tempfile location, then I think there's no point in storing it in the first place, is there?)

This now also conflicts with #155 which also changed this and is now merged. Needs to be resolved.

Reply:
I solved the conflict by keeping your changes. I agree that a temporary folder doesn't make much sense here.